### DISCUSSION TITLE
日本サーバーベース FP pickup関連のスクリーンショット

### DISCUSSION THREAD

**Discussion by foxlesbiao** [MaaAssistantArknights/MaaAssistantArknights#1329] [Show and tell]
以下は、その時のスクリーンショットです。
[JP server ベース.zip](https://github.com/MaaAssistantArknights/MaaAssistantArknights/files/9180187/JP.server.zip)

**Comment by MistEO** [comment]
Thanks for your help! we will implement these features in JP client as soon as possible. ✿✿ヽ(°▽°)ノ✿

**Comment by foxlesbiao** [comment]
> Thanks for your help! we will implement these features in JP client as soon as possible. ✿✿ヽ (°▽°) ノ✿

ノーサンキュー、不足しているものがあれば、ご連絡ください。

**Comment by foxlesbiao** [comment]
!![image](https://user-images.githubusercontent.com/41581909/180786160-4571de17-019b-464b-98bd-0c949128a7e3.png)

このサークル内のモンスターは、幹部を配置しても破壊されない

**Comment by foxlesbiao** [comment]
![image](https://user-images.githubusercontent.com/41581909/180787521-94792c12-1a50-40b3-8029-7fa6d703c08b.png)
![image](https://user-images.githubusercontent.com/41581909/180787566-57a73a60-33c2-4860-b85a-7161d7f17a77.png)
このレベルも同じです

**Comment by MistEO** [comment]
The automatic roguelike function is not too intelligent, the main purpose is to obtain candles (EXP) stably and reduce manual repetitive labor. 
In the future, a custom formation function may be developed to make it more interesting, ~~but still not smart~~

**Comment by foxlesbiao** [comment]
> The automatic roguelike function is not too intelligent, the main purpose is to obtain candles (EXP) stably and reduce manual repetitive labor. In the future, a custom formation function may be developed to make it more interesting, ~but still not smart~

なるほど、わかりました。

**Comment by lhhxxxxx** [comment]
> > The automatic roguelike function is not too intelligent, the main purpose is to obtain candles (EXP) stably and reduce manual repetitive labor. In the future, a custom formation function may be developed to make it more interesting, ~but still not smart~
> 
> なるほど、わかりました。

ここのロジックは、特定ポイントの周辺に幹部を配置するだけです。。。

**Comment by MistEO** [comment]
Maybe better as a discussion.
